src/OSPSuite.Core/Comparison/ExpressionProfileDiffBuilder.cs

```diff
@@ -0,0 +1,30 @@
+﻿using OSPSuite.Core.Domain.Builder;
+
+namespace OSPSuite.Core.Comparison
+{
+   public class ExpressionProfileBuildingBlockDiffBuilder : BuildingBlockDiffBuilder<ExpressionProfileBuildingBlock, ExpressionParameter>
+   {
+      public ExpressionProfileBuildingBlockDiffBuilder(ObjectBaseDiffBuilder objectBaseDiffBuilder, EnumerableComparer enumerableComparer) : base(objectBaseDiffBuilder, enumerableComparer)
+      {
+
+      }
+
+      public override void Compare(IComparison<ExpressionProfileBuildingBlock> comparison)
+      {
+         base.Compare(comparison);
+         CompareValues(x => x.PKSimVersion, x => x.PKSimVersion, comparison);
+         CompareStringValues(x => x.MoleculeName, x => x.MoleculeName, comparison);
```

Comment:
Molecule Name, Species, Category are all derived from the name so it's probably not required because it will be redundant. Name is also probably compared in the base class
Last there is a settings that is used. (compare relevant properties or whatever). PKSim Version is probably not one of them. Type ? not sure

Reply:
removed Molecule Name, Species, Category and Name, will test the comparison to see that everything still works.
@msevestre I am not sure wha tyou mean by the last part:

> Molecule Name, Species, Category are all derived from the name so it's probably not required because it will be redundant. Name is also probably compared in the base class
Last there is a settings that is used. (compare relevant properties or whatever). PKSim Version is probably not one of them. Type ? not sure